orga/app/routes/authenticated/students/list.js

```diff
@@ -18,13 +17,15 @@ export default class ListRoute extends Route {
       filter: {
         organizationId: this.currentUser.organization.id,
         lastName: params.lastName,
+        firstName: params.firstName,
       },
     });
   }
 
   resetController(controller, isExiting) {
     if (isExiting) {
       controller.set('lastName', null);
+      controller.set('firstName', null);
```

Comment:
On peut en profiter pour "octaniser". Il y a fort à parier que
```suggestion
      controller.lastName = null;
      controller.firstName = null;
```
ça marche bien

Reply:
Ben ça marche pas :(